src/components/modals/AddOrEditMovieForm/Select.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import { FormikErrors, FormikTouched } from "formik";
+import styles from "../../../css-modules/modal.module.css";
+
+// Interfaces
+import Movie from "../../../types/movie.interface";
+
+interface Props {
+    id: string,
+    title: string,
+    info: Movie,
+    value: keyof Movie,
+    errors: FormikErrors<Movie>,
+    touched: FormikTouched<Movie>,
+    handleChange: React.ChangeEventHandler<HTMLSelectElement>,
+    handleBlur: React.FocusEventHandler<HTMLSelectElement>
+}
+
+function Select(props: Props) {
+  // Props Extraction
+  const {
+    id, title, info, value, errors, touched, handleChange, handleBlur,
+  } = props;
+
+  return (
+    <div className={styles.inputGroupLarge}>
+      <label htmlFor={id} className={styles.label}>{ title }</label>
+      <select
+        id={id}
+        name={value}
+        value={info[value]}
+        className={errors[value] && touched[value] ? styles.inputWithError : styles.input}
+        multiple
+        onChange={handleChange}
+        onBlur={handleBlur}
+      >
+        <option value="Action">Action</option>
```

Comment:
The better approach here will be to create a constant with array of options and then iterate through it.
```js
const OPTIONS = [""Advanture", ..., "Crime"];

...

OPTIONS.map(option => <option key={option} value={option}>{option}</option>)
```

Reply:
resolved